astropy/stats/sigma_clipping.py

```diff
@@ -590,7 +600,7 @@ def __call__(self, data, axis=None, masked=True, return_bounds=False,
                 return np.ma.filled(data.astype(float), fill_value=np.nan)
 
         # Shortcut for common cases where a fast C implementation can be used.
-        if self.cenfunc in ('mean', 'median') and self.stdfunc == 'std' and axis is not None and not self.grow:
+        if self.cenfunc in ('mean', 'median') and self.stdfunc in ('std', 'mad_std') and axis is not None and not self.grow:
```

Comment:
break line?

Reply:
Yes, please.  The flake8 test uses a maximum line length of 100.  This line is longer than that.

Reply:
Done